code/_onclick/item_attack.dm

```diff
@@ -137,6 +137,9 @@
 	if(I.force)
 		var/justice_mod = 1 + (get_modified_attribute_level(user, JUSTICE_ATTRIBUTE)/100)
 		apply_damage((I.force * justice_mod), I.damtype, white_healable = TRUE)
+		if(istype(I, /obj/item/ego_weapon))
+			var/obj/item/ego_weapon/theweapon = I
+			apply_damage((I.force * (theweapon.force_multiplier - 1)), I.damtype, white_healable = TRUE)
```

Comment:
This can result in double messages, might simply be better to make another variable for the damage that's being done and have this if-statement modify that instead.
```
var/damage = I.force * justice_mod
if(istype(I, /obj/item/ego_weapon))
	var/obj/item/ego_weapon/theweapon = I
	damage += I.force * (theweapon.force_multiplier - 1)
apply_damage(damage, I.damtype, white_healable = TRUE)
```